doc/reference/reference_lua/box_space/pairs.rst

```diff
@@ -10,61 +10,92 @@ space_object:pairs()
 
         Search for a tuple or a set of tuples in the given space, and allow
         iterating over one tuple at a time.
+        To search by the specific index, use the :ref:`box_index-pairs` method.
 
         :param space_object space_object: an :ref:`object reference
                                           <app_server-object_reference>`
         :param scalar/table key: value to be matched against the index key,
                                  which may be multi-part
-        :param         iterator: see :doc:`/reference/reference_lua/box_index/pairs`
+        :param         iterator: the :ref:`iterator type <box_index-iterator-types>`. The default iterator type is 'EQ'
+        :param after: a tuple or the position of a tuple (:ref:`tuple_pos <box_index-tuple_pos>`) after which ``pairs`` starts the search. You can pass an empty string or :ref:`box.NULL <box-null>` to this option.
 
-        :return: `iterator <https://www.lua.org/pil/7.1.html>`_ which can be
+        :return: The `iterator <https://www.lua.org/pil/7.1.html>`_, which can be
                  used in a for/end loop or with `totable()
-                 <https://luafun.github.io/reducing.html#fun.totable>`_
+                 <https://luafun.github.io/reducing.html#fun.totable>`_.
 
         **Possible errors:**
 
-        *    No such space.
-        *    Wrong type.
+        *   no such space
```

Comment:
Sentence style capitalization and punctuation is used for the same text in the next file.